assets/javascripts/application.js

```diff
@@ -2,25 +2,33 @@
 //= require_directory ./plugins/
 //= require_self
 
-$(document).ready(function() {
+$(document).ready(function () {
+  branchCoverageActivated = $("body").data("branch-coverage") == true;
+
   // Configuration for fancy sortable tables for source file groups
+  lineColumns = [null, { "sType": "percent" }, null, null, null, null, null];
+  branchColumns = [{ "sType": "percent" }, null, null, null];
```

Comment:
Why does only one column get the type defined? If there is a reason it would be good to document it

Reply:
That's because branch columns has only one (first one) column with percentage, other ones are just counters: https://github.com/colszowka/simplecov-html/blob/master/views/file_list.erb#L67

Reply:
Might be good to comment here to look into the "file_list.erb" to checkout which columns are used there.

Reply:
Yeah good idea, I might have a look at jquery data tables if we could give the others a type or at least document that null has correct count/integer sorting. Good spot!

Reply:
Our datatables version is from... 2010. I can't find a manual/reference with a quick look and it's probably not worth a long look.